### DISCUSSION TITLE
SAP AND MS SQL

### DISCUSSION THREAD

**Discussion by lucky277** [utmstack/UTMStack#884] [Q&A] [answered]
Good day,

Please advise if utmstack can perform security monitoring for SAP and MS SQL

**Comment by c3s4rfred** [comment, marked as answer]
Hi @lucky277, UTMStack don't have an specific integration for those technologies, but, if those technologies can send logs through any of the integrations available (maybe json-input or syslog). Then you can create rules to monitor the logs.

Best regards

**Comment by lucky277** [comment]
Hi Freddy,

Thank you for the response.

I have tried but i can't get logs for SAP or MS SQL

**Reply by c3s4rfred** [reply]
Hi @lucky277 you have to check your SAP or MS SQL syslog configurations, or check that the required ports according to the selected integration, are open in the agent manager's machine.

Best regards
